### PR TITLE
SAK-50607 Gradebook Tabulator error on console when adding Item

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-50607

### CODE REVIEW

**Comment by ottenhoff** [comment]
please rebase and resolve conflicts!

File: library/src/skins/default/src/sass/modules/tool/gradebook/_gradebook.scss [hunk 1]
```diff
@@ -1738,3 +1738,23 @@ max-height:100px;
...
+
+.gb-cell-selected:after{
+  content: '';
+  position: absolute;
+  right: -3px;
+  bottom: -3px;
+  width: 6px;
+  height: 6px;
+  background-color: #a5dfa7;
+  border-radius: 999px;
```
Comment (ottenhoff) [inline library/src/skins/default/src/sass/modules/tool/gradebook/_gradebook.scss:1759]: that's a big border-radius!

Comment (kunaljaykam) [reply library/src/skins/default/src/sass/modules/tool/gradebook/_gradebook.scss:1759]: We don't need that. I will get rid of that. Thanks

**Review by ottenhoff — APPROVED** [review]

**Comment by kunaljaykam** [comment]
@ottenhoff merge this please

**Pull request merged**